Gems/Maestro/gem.json

```diff
@@ -20,6 +20,10 @@
     "requirements": "",
     "documentation_url": "https://www.o3de.org/docs/user-guide/gems/reference/animation/maestro/",
     "dependencies": [
-        "LmbrCentral"
+        "LmbrCentral",
+        "Atom",
+        "Atom_Bootstrap",
+        "Atom_RHI",
+        "Atom_RPI"
```

Comment:
(No change required here, just taking a note) 

Sure is a pity that we have to depend on all of these, but I suppose the monolithic nature of Maestro makes it impossible for Atom to have a "maestro plugin" instead of having maestro have an atom plugin.